### ISSUE TITLE
Realm gating

### ISSUE THREAD

**Issue by taggrx** [TaggrNetwork/taggr#109]
Ideas:

- filter users by min karma
- approval based membership

**Comment by mechaquan** [comment]
I agree with Realm gating... see my post on RealmDAO... I think we need to think about what realms are or could be.

I'd like to see some additional economics around realms too... subscription based realms... I think Grayghost and the Macro realm would be a good test use case for this.
